draft-ietf-acme-scoped-dns-challenges.mkd

```diff
@@ -253,7 +173,9 @@ As this challenge creates strong dependency on the `kid` account identifier, the
 
 These challenges follow the recommendations set out in "Domain Control Validation using DNS" {{I-D.draft-ietf-dnsop-domain-verification-techniques}} for supporting multiple intermediates within the context of {{!RFC8555}}.
 
-In both `dns-account-01` and `dns-02`, the same security considerations apply for the integrity of authorizations ({{!RFC8555, Section 10.2}}) and DNS security ({{!RFC8555, Section 11.2}}) as in the original specification for `dns-01`. They both differ from `dns-01` in that the challenges are scoped to the particular name being validated without relying upon CAA ({{!RFC8659}}).
+The new `dns-account-01` challenge shares the same security considerations as the original `dns-01` specification regarding the integrity of authorizations {{!RFC8555, Section 10.2}} and DNS security {{!RFC8555, Section 11.2}}.
+
+The `dns-account-01` challenge differs from the original specification by scoping the challenge to the specific name being validated, without relying on CAA records {{!RFC8659}}. This modification enables an ACME client to verify and validate an ACME server's operations with respect to the DNS names it queries.
```

Comment:
> This modification enables an ACME client to verify and validate an ACME server's operations with respect to the DNS names it queries.

This alone does not verify the server is performing correctly and only grants authorization to the intended identifier or performs its job ultimately correctly according to policy? I am not sure I fully understand the benefit.

Reply:
I think we could pull/reference language from the "best practice" draft depending on what we keep.